templates/field--field-funding-contact-email.html.twig

```diff
@@ -0,0 +1,5 @@
+<div class="funding-ct--summary-section--contact-info-email">
+    {% for item in items %}
+        <span class="wdn-icon-mail" aria-hidden="true"></span><span class="wdn-text-hidden">email icon</span>{{ item }}
```

Comment:
This field only permits one value at this time. Changing expression to use index 0 instead.